### PR TITLE
gaming section text added on tumbleweed page

### LINKED ISSUE #5: Add some points about gaming to Tumbleweed
We have had complaints from gaming community that we don't do enough to tell people Leap might be too old for gaming, so we should make it clear that Tumbleweed is for gamers

### PR DESCRIPTION
**Pull request by sharabhh**
Fixed #5 
Added some information about gaming on Tumbleweed.

Before: 
image

After:
image

### CODE REVIEW

**Comment by hellcp** [comment]
Hm, this PR contains no changes that are shown in the screenshot and just changes the dependencies instead

**Comment by hellcp** [comment]
Sorry, I assumed you figured out how to do this properly, this is where the features are defined: https://github.com/openSUSE/get-o-o/blob/main/_data/locales/en.yml#L357, you just need to modify this file

**Comment by sharabhh** [comment]
> Sorry, I assumed you figured out how to do this properly, this is where the features are defined: https://github.com/openSUSE/get-o-o/blob/main/_data/locales/en.yml#L357, you just need to modify this file

ohh okay I'll modify this one

**Comment by sharabhh** [comment]
image

**Comment by sharabhh** [comment]
@hellcp will this work? or should I create a separate pull request with only this file edited. Right now I think my previous commits are also showing up.

**Comment by hellcp** [comment]
They are, you can use `git rebase -i HEAD~3`, and mark the commits in the file that shows up appropriately based on the description in that file and then `git log` to verify you only have the commits you have on the branch and then `git push --force` to push the commits you want to this PR

File: _data/locales/en.yml [hunk 1]
```diff
@@ -391,3 +391,7 @@ Tumbleweed_features:
     name: Stands on Firm Ground
     description: |
       Tumbleweed builds on decades of usage, testing and debugging by hundreds of power-users, developers, system administrators and demanding doers that cannot afford to jeopardize their workflow. Tumbleweed's solidity is embodied in many core packages whose DNA stems from the venerable SUSE Linux Enterprise Server.
+  gaming:
+    name: Gaming
+    description: |
+     Tumbleweed follows a rolling release model, providing users with the latest software updates and features, including gaming-related packages. OpenSUSE Tumbleweed is designed to maintain stability, ensuring a reliable gaming experience. In short, you can expect epic gaming performance with stability.
```
Comment (tacerus) [inline _data/locales/en.yml:397]: `OpenSUSE` -> `openSUSE`

See https://en.opensuse.org/Help:Style#openSUSE_spelling.

Comment (sharabhh) [reply _data/locales/en.yml:397]: oh okay, I'll fix this spelling.

File: _data/locales/en.yml [hunk 2]
```diff
@@ -391,3 +391,7 @@ Tumbleweed_features:
     name: Stands on Firm Ground
     description: |
       Tumbleweed builds on decades of usage, testing and debugging by hundreds of power-users, developers, system administrators and demanding doers that cannot afford to jeopardize their workflow. Tumbleweed's solidity is embodied in many core packages whose DNA stems from the venerable SUSE Linux Enterprise Server.
+  gaming:
+    name: Gaming
+    description: |
+     Tumbleweed follows a rolling release model, providing users with the latest software updates and features, including gaming-related packages. openSUSE Tumbleweed is designed to maintain stability, ensuring a reliable gaming experience. In short, you can expect epic gaming performance with stability.
```
Comment (hellcp) [inline _data/locales/en.yml:397]: We already explained stability and and latest packages being available above, let's instead talk about why those things matter specifically for gaming
```suggestion
     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
```

File: _includes/icons/gaming.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```
Comment (hellcp) [inline _includes/icons/gaming.svg:1]: We tend to use remix icons for this, how about using them? https://remixicon.com/

File: _data/locales/en.yml (hunk 2 above)
Comment (sharabhh) [reply _data/locales/en.yml:397]: okay, I'll write more specific to gaming

Comment (sharabhh) [reply _data/locales/en.yml:397]: should I push your suggested text or should I add a few more lines? @hellcp

Comment (hellcp) [reply _data/locales/en.yml:397]: I don't know, I'm fine with mine, but feel free to expand it

File: _includes/icons/gaming.svg (hunk 3 above)
Comment (sharabhh) [reply _includes/icons/gaming.svg:1]: Okay

**Comment by sharabhh** [comment]
@hellcp I changed it to remix icons

**Comment by hellcp** [comment]
The Gemfile.lock manifested in this, so try removing it from the commit

**Pull request closed without merging**